shuup/core/fields/__init__.py

```diff
@@ -25,6 +25,8 @@
 
 IdentifierValidator = RegexValidator("[a-z][a-z_]+")
 
+DECIMAL_FIELD_DECIMAL_PLACES = 9
```

Comment:
Is this being added to _all_ decimal fields we are using? if not, use lower level values too `MONEY_FIELD_DECIMAL_PLACES = 9´ etc. If it's being used everywhere, nevermind.